rpc/handlers.go

```diff
@@ -0,0 +1,40 @@
+package rpc
+
+import (
+	"encoding/hex"
+	"log"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/pokt-network/pocket/app"
+)
+
+func (s *rpcServer) GetV1Health(ctx echo.Context) error {
+	return ctx.NoContent(http.StatusOK)
+}
+
+func (s *rpcServer) GetV1Version(ctx echo.Context) error {
+	return ctx.String(http.StatusOK, app.AppVersion)
+}
+
+func (s *rpcServer) PostV1ClientBroadcastTxSync(ctx echo.Context) error {
+	params := new(RawTXRequest)
+	if err := ctx.Bind(params); err != nil {
+		return ctx.String(http.StatusBadRequest, "bad request")
+	}
+	bz, err := hex.DecodeString(params.RawHexBytes)
+	if err != nil {
+		return ctx.String(http.StatusBadRequest, "cannot decode tx bytes")
+	}
+	height := s.bus.GetConsensusModule().CurrentHeight()
```

Comment:
NIT: `s.GetBus()`, here and below

Reply:
`rpcServer` is not an `IntegratableModule`. We are just accessing the `bus` that's a private reference.

Do you prefer if it implements GetBus and SetBus?